budget/budgetRouter.js

```diff
@@ -0,0 +1,20 @@
+const express = require('express');
+
+const Budgets = require('./budgetModel');
+
+const router = express.Router();
+
+router.post('/', (req, res) => {
+    const budgetInfo = req.body;
+    const budget = new budget(budgetInfo);
```

Comment:
this all looks correct except I see where you imported Budgets but i dont see where budget is being imported